crypto/asn1/a_time.c

```diff
@@ -70,6 +70,11 @@ static void determine_days(struct tm *tm)
     /* Zeller's congruence */
     tm->tm_wday = (d + (13 * m) / 5 + y + y / 4 + c / 4 + 5 * c + 6) % 7;
 }
+static int ascii_isdigit(char inchar) {
+  if (inchar > 0x2F && inchar < 0x3A)
```

Comment:
Style nit: Please use 4 spaces for indentation

Reply:
Done